modules/acs-requirements.adoc

```diff
@@ -5,10 +5,22 @@
 [id="acs-general-requirements_{context}"]
 = General requirements
 
-ifeval::["{context}" == "install-rhacs-ocp"]
+ifeval::["{context}" == "prerequisites-ocp"]
 :openshift:
 endif::[]
 
+ifeval::["{context}" == "prerequisites-other"]
```

Comment:
@gaurav-nelson Sorry missed this the first time through, but you do not close these ifeval directives at the end of the file.